packages/synapse-sdk/src/piece/piece.ts

```diff
@@ -7,6 +7,7 @@
 import type { LegacyPieceLink as LegacyPieceCIDType, PieceLink as PieceCIDType } from '@web3-storage/data-segment'
 import * as Hasher from '@web3-storage/data-segment/multihash'
 import { Unpadded } from '@web3-storage/data-segment/piece/size'
+import { ethers } from 'ethers'
```

Comment:
going forward I think we're preferring Viem variants of all of these, but this should ok for now